app/models/search/tag_query.rb

```diff
@@ -77,8 +78,19 @@ def type_filter
     { term: { tag_type: options[:type] } } if options[:type]
   end
 
-  def canonical_filter
-    term_filter(:canonical, bool_value(options[:canonical])) if options[:canonical].present?
+  def wrangling_status_filter
+    case options[:wrangling_status]
+    when "canonical"
+      term_filter(:canonical, true)
+    when "noncanonical"
+      term_filter(:canonical, false)
+    when "synonymous"
+      [{ exists: { field: "merger_id" } }, term_filter(:canonical, false)]
```

Comment:
Can you add a helper method `exists_filter(field, value)` in the Query class, similar to `term_filter`?

You can use the new helper here as well:

https://github.com/otwcode/otwarchive/blob/25865ac2f122e8027994be6809a54bacdfdf9d6e/app/models/search/tag_query.rb#L133